test/interest-rate-test.ts

```diff
@@ -157,11 +109,166 @@ describe('interest rates', function () {
     // totalBorrowBase / totalSupplyBase
     // = 0 / 100 = 0
     expect(utilization).to.be.equal(0);
-    // (interestRateBase + interestRateSlopeLow * utilization) * utilization * (1 - reserveRate)
-    // = (0.005 + 0.1 * 0) * 0 * 0.9 = 0
+    // interestRateBase + interestRateSlopeLow * utilization
+    // = 0 + 0.04 * 0 = 0
     assertInterestRatesMatch(0, supplyRate.mul(SECONDS_PER_YEAR));
     // interestRateBase + interestRateSlopeLow * utilization
-    // = 0.005 + 0.1 * 0 = 0.005
-    assertInterestRatesMatch(exp(5, 15), borrowRate.mul(SECONDS_PER_YEAR));
+    // = 0.01 + 0.05 * 0 = 0.01
+    assertInterestRatesMatch(exp(0.01, 18), borrowRate.mul(SECONDS_PER_YEAR));
   });
 });
+
+// === Tests for old kink model with fixed reserve rate (v2 model) ===
```

Comment:
We want to keep this here?

Reply:
I'm wondering if we should store both versions of Comet/Configurator in the repo, in case we ever want to switch between the two rate models. Though, it does seem difficult managing multiple versions of Comet/Configurator.

Reply:
We can always use git?

Reply:
Maybe a good time to drop a tag

Reply:
So we have a few impl contracts lying around and the one Comet extends from at any given time is the one we use? Wonder if this would get hard to manage and we should always just think about variant Comets using git commits/history, since they can vary in other ways besides IR models too

Reply:
Actually, the dependencies are the other way around. I was thinking something like this:

```
Contracts in repo:
CometOldModel.sol: CometOldModel extends CometBase
CometNewModel.sol: CometNewModel extends CometBase
CometXModel.sol: CometXModel extends CometBase
CometBase.sol: Basically an abstract Comet.sol but with the interest rate functions defined as abstract 
```

That way, the diffs among these contracts are only the interest model implementations. It'll make it easier to maintain different types of interest rate models IMO. But yeah, just thinking out loud here 🧠

Reply:
Removed the commented-out tests for the old IR model.